sklearn/linear_model/tests/test_coordinate_descent.py

```diff
@@ -350,10 +351,39 @@ def test_enet_cv_positive_constraint():
     assert_true(min(enetcv_constrained.coef_) >= 0)
 
 
+def test_uniform_targets():
+    enet = ElasticNetCV(fit_intercept=True)
+    m_enet = MultiTaskElasticNetCV(fit_intercept=True)
+    lasso = LassoCV(fit_intercept=True)
+    m_lasso = MultiTaskLassoCV(fit_intercept=True)
+
+    models_single_task = (enet, lasso)
+    models_multi_task = (m_enet, m_lasso)
+
+    rng = np.random.RandomState(0)
+
+    X_train = rng.random_sample(size=(10, 3))
+    X_test = rng.random_sample(size=(10, 3))
+
+    y1 = np.empty(10)
+    y2 = np.empty((10, 2))
+
+    for model in models_single_task:
+        for y_values in (0, -5, 5, 4.5, -4.5, 10000):
+            y1.fill(y_values)
+            assert_array_equal(model.fit(X_train, y1).predict(X_test), y1)
+
+    for model in models_multi_task:
+        for y_values in (0, -5, 5, 4.5, -4.5, 10000):
+            y2[:, 0].fill(y_values)
+            y2[:, 1].fill(2 * y_values)
+            assert_array_equal(model.fit(X_train, y2).predict(X_test), y2)
```

Comment:
> Around 2 seconds... Is that acceptable?

It would be nice if it were smaller. There are thousands of tests in
scikit-learn. It would not be possible for all to take 2s.